pkg/basicstation/cups/update_info.go

```diff
@@ -179,17 +219,37 @@ func (s *Server) UpdateInfo(c echo.Context) error {
 	}
 
 	res := UpdateInfoResponse{}
```

Comment:
Nitpick: IIRC, we preferred `var res UpdateInfoResponse{}`

Reply:
Nope.

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/pull/1077#discussion_r309272681

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/pull/1077#discussion_r309619724

Reply:
🤦‍♂ I'm always confused with this crap. We really should stop caring so much about equivalent code.

Reply:
I agree with you on this. I'm just repeating what was said to me in my PRs.

Reply:
I'm also fine with making it all more consistent: https://github.com/TheThingsNetwork/lorawan-stack/compare/issue/1313-bs-cups...fix/var-declares

Maybe better to do that in a separate PR then...

Reply:
👏